packages/hub-extension/src/index.ts

```diff
@@ -100,7 +100,7 @@ function activateHubExtension(
  */
 const hubExtension: JupyterFrontEndPlugin<void> = {
   activate: activateHubExtension,
-  id: 'jupyter.extensions.hub-extension',
+  id: '@jupyterlab/hub-extension',
```

Comment:
Maybe we could rename this plugin to `@jupyterlab/hub-extension:plugin` directly?